### DISCUSSION TITLE
Building tecnativa/doodba:17.0-onbuild on WSL2

### DISCUSSION THREAD

**Discussion by jmartinacu** [Tecnativa/doodba#620] [Q&A] [answered]
Hello,

I am trying to execute doodba/17.0.Dockerfile on a windows machine  using WSL2.

When I get to the next step of the Dockerfile, the build stops and does not do any other steps. Which is when this command is executed:

![image](https://github.com/user-attachments/assets/74d6383a-38eb-4c1c-b4ab-a9fe6aaaf77f)

Is there any way to do the build of this container using WSL2 or is it only compliant on Linux systems.

**Comment by josep-tecnativa** [comment]
Maybe there is some issue by creating groups on WSL2. Don't you have any error message?

**Comment by ndemarco** [comment]
To specifically answer your question, yes; I am running Doodba in WSL2 right now, and every day, so it definitely works. In fact, it works remarkably well with WSL2.

Please share more about what you are doing to get to the point where this fails. Are you running `invoke develop`? What Linux distro have you installed in WSL2?

**Comment by jmartinacu** [comment]
I am trying to build a docker image from tecnativa/doodba:17.0-onbuild on windows 11 using WSL.

My configuration of WSL is the following:

![image](https://github.com/user-attachments/assets/461f5d55-0a4d-41ed-9c24-94d5cb438021)

With the command ```docker build -t my_odoo .``` using this Dockerfile:

![image](https://github.com/user-attachments/assets/3147cedc-76c9-4781-9654-19f78da218c7)

When I execute the command to build the docker image this is the output:

<img width="794" alt="image" src="https://github.com/user-attachments/assets/19b09d36-0cd9-4dec-b87c-d57eb3be4522">

It stops on the step ```RUN groupadd``` and doesn't do anything.

This is the info about the docker command I am currently using:

![image](https://github.com/user-attachments/assets/90c6b97f-f699-4e87-bd3c-20d001143083)

**Comment by ndemarco** [comment]
Is there a reason you are running `docker build` explicitly instead of running `invoke develop` along with the other `invoke` commands listed in the documentation for getting started with doodba-copier-template?

Start with the basic deployment for the development environment. You have Docker running properly already. Make sure your [dependencies are installed correctly](https://www.youtube.com/watch?v=XwXrkbPzn_M&list=PLPC4k8hMtfG-ChkkfphVgtUDFyayISeEx).

After all of the above, follow [this set of instructions](https://github.com/Tecnativa/doodba-copier-template/blob/main/docs/daily-usage.md#development).

**Reply by jmartinacu** [reply, marked as answer]
Hello, thanks for the answer.

I could build the image after installing, in the WSL2 machine, all the dependencies needed. Which are listed in the `Installation and Usage` section of the README.
